Source/projections/Store/ProjectionStore.ts

```diff
@@ -51,6 +53,21 @@ export class ProjectionStore extends IProjectionStore {
         super();
     }
 
+    /** @inheritdoc */
+    of<TProjection>(type: Constructor<TProjection>): IProjectionOf<TProjection>;
+    of<TProjection>(type: Constructor<TProjection>, projection: ProjectionId | Guid | string): IProjectionOf<TProjection>;
+    of<TProjection>(type: Constructor<TProjection>, projection: ProjectionId | Guid | string, scope: ScopeId | Guid | string): IProjectionOf<TProjection>;
+    of<TProjection>(type: Constructor<TProjection>, maybeProjection?: ProjectionId | Guid | string, maybeScope?: ScopeId | Guid | string): IProjectionOf<TProjection> {
+        if (!maybeProjection) {
```

Comment:
We haven't really uses this kind of falsey check elsewhere. A little more explicit:
```suggestion
        if (maybeProjection === undefined) {
```